src/Controls/src/Core/Label/Label.cs

```diff
@@ -382,6 +382,17 @@ void ITextElement.OnTextColorPropertyChanged(Color oldValue, Color newValue)
 		void ITextElement.OnCharacterSpacingPropertyChanged(double oldValue, double newValue) =>
 			InvalidateMeasureIfLabelSizeable();
 
+		TextDecorations ILabel.TextDecorations
+		{ 
+			get
+			{
+				if (Text is not null)
```

Comment:
Is this condition here for some not evident reason?

Reply:
Whoops :-/ fixed